model/config.go

```diff
@@ -47,6 +47,7 @@ const (
 	GENERIC_NO_CHANNEL_NOTIFICATION = "generic_no_channel"
 	GENERIC_NOTIFICATION            = "generic"
 	GENERIC_NOTIFICATION_SERVER     = "https://push-test.mattermost.com"
+	MM_SUPPORT_ADDRESS              = "catalin@mattermost.com" //support@mattermost.com CITOMAI
```

Comment:
@catalintomai can you revert the support email address, as long as you are not willing to handle all the support requests :). Currently its sending the email to your email account.

Reply:
@srkgupta - sure, will change it before submission = ).